### PR TITLE
Add CI configuration files

### PR DESCRIPTION
**Pull request by martijnversluis**

### CODE REVIEW

File: templates/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        dockerfile {
+          filename "dockerfiles/ci/Dockerfile"
+          additionalBuildArgs support.ciDockerFileBuildArgs()
+          args "${support.ciDockerFileRunArgs(projectId)} -u root"
+        }
+      }
+
+      environment {
+        RAILS_ENV = 'test'
+        CODECLIMATE_REPO_TOKEN = credentials('ethnicchic_codeclimate_test_coverage_reporter_id')
```
Comment (drumusician) [inline templates/Jenkinsfile]: ethnicchic? 🏃

File: templates/README.md [hunk 2]
```diff
@@ -19,7 +19,8 @@ For specific version numbers please refer to
 - create Dokku repository
 - add this project to Jenkins
 - [add the project to CodeClimate](https://codeclimate.com/accounts/5aa242dd800550027b000371/add_vcs_repo/new)
-- create Jenkins secret containing the CodeClimate test coverage reporter ID
+- create Jenkins credential `ethnicchic_codeclimate_test_coverage_reporter_id` containing the CodeClimate test
```
Comment (drumusician) [inline templates/README.md]: 😱

File: templates/dockerfiles/ci/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+RUN apt-get update -qq && apt-get install -y \
+    build-essential \
+    libpq-dev \
+    curl \
+    postgresql \
+    postgresql-contrib \
+    google-chrome-stable \
+    unzip \
+    oracle-java8-installer \
+    elasticsearch \
```
Comment (drumusician) [inline templates/dockerfiles/ci/Dockerfile]: java8 and elasticsearch needed?

**Comment by martijnversluis** [comment]
@drumusician Could you check again?

File: templates/Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        sh 'git push dokku-production HEAD:master --force'
+      }
+      post {
+        always {
+          script { support.slackNotification(channel: "#[NOTIFICATION_CHANNEL]") }
+        }
+      }
+    }
+  }
+
```
Comment (nickve28) [inline templates/Jenkinsfile]: unneccessary empty lines

**Pull request merged**